iTunes Media App/Views/SongTableViewCell.swift

```diff
@@ -10,9 +10,72 @@ import Foundation
 import UIKit
 
 class SongTableViewCell: UITableViewCell {
-    // instance property means you NEED an instantiated object to use it
-    // use static when you aren't creating the object just yet
     static let reuseID = String(describing: self)
     
+    private lazy var songNameLabel: UILabel = {
+        let lbl = UILabel()
```

Comment:
Avoid abbreviations. 
```suggestion
        let label = UILabel()
```